packages/components/basic-table/stories/basic-table.happo.js

```diff
@@ -101,7 +101,7 @@ storiesOf('ts-basic-table', module).add('test', () => {
 						mobileDescription: 'Tradeshift | 5 participants',
 						type: 'document'
 					},
-					type: { status: 'ok', text: 'External' },
+					type: { status: 'success', text: 'External' },
```

Comment:
Could you please change the type status "error" to "warning" in the next row object? So it will look the same as it was and there will be no diffs in happo.

Reply:
sure 👍